system-addon/content-src/styles/_theme.scss

```diff
@@ -0,0 +1,80 @@
+// Default theme
+.outer-wrapper {
+  --newtab-background-color: $grey-10;
+  --newtab-text-primary-color: $grey-90;
+  --newtab-text-secondary-color: $grey-50;
+  --newtab-text-conditional-color: $grey-60;
+  --newtab-link-primary-color: $blue-60;
+  --newtab-link-secondary-color: $teal-70;
+  --newtab-border-primary-color: $grey-40;
+  --newtab-border-secondary-color: $grey-30;
+  --newtab-icon-primary-color: $grey-90-80;
+  --newtab-icon-secondary-color: $grey-90-60;
+  --newtab-icon-tertiary-color: $grey-30;
+  --newtab-textbox-border: $grey-90-20;
+  --newtab-textbox-color: inherit;
+  --newtab-button-primary-color: $blue-60;
+  --newtab-button-secondary-color: inherit;
+  --newtab-contextmenu-button-color: $white;
+  --newtab-modal-color: $white;
+  --newtab-overlay-color: $grey-20;
+  --newtab-searchbox-color: $grey-10;
+  --newtab-searchbox-border-color: transparent;
+  --newtab-searchbox-text-color: inherit;
+  --newtab-searchbox-icon-color: $grey-90-40;
+  --newtab-section-header-text-color: $grey-50;
+  --newtab-section-navigation-text-color: $grey-50;
+  --newtab-section-navigation-link-color: $blue-40;
+  --newtab-section-active-color: $grey-20-60;
+  --newtab-section-active-contextmenu-color: $grey-90;
+  --newtab-topsites-label-color: inherit;
+  --newtab-card-background-color: $white;
+  --newtab-inner-box-shadow-color: $black-10;
+}
+
+// Dark theme
+.outer-wrapper.dark-theme {
+  --newtab-background-color: $grey-80;
+  --newtab-text-primary-color: $grey-10;
+  --newtab-text-secondary-color: $grey-10-40;
+  --newtab-text-conditional-color: $grey-10;
+  --newtab-link-primary-color: $blue-40;
+  --newtab-link-secondary-color: $blue-40;
+  --newtab-border-primary-color: $grey-10-80;
+  --newtab-border-secondary-color: $grey-10-40;
+  --newtab-icon-primary-color: $grey-10-80;
+  --newtab-icon-secondary-color: $grey-10-40;
+  --newtab-icon-tertiary-color: $grey-10-20;
+  --newtab-textbox-border: $grey-10-40;
+  --newtab-textbox-color: $grey-80;
+  --newtab-button-primary-color: $blue-60;
+  --newtab-button-secondary-color: $grey-70;
+  --newtab-contextmenu-button-color: $grey-80;
+  --newtab-modal-color: $grey-90;
+  --newtab-overlay-color: $grey-80;
+  --newtab-searchbox-color: $grey-60;
+  --newtab-searchbox-border-color: $grey-10-40;
+  --newtab-searchbox-text-color: $grey-10-60;
+  --newtab-searchbox-icon-color: $grey-10-60;
+  --newtab-section-header-text-color: $grey-10-80;
+  --newtab-section-navigation-text-color: $grey-10-80;
+  --newtab-section-active-color: $grey-60;
+  --newtab-section-active-contextmenu-color: $white;
+  --newtab-topsites-label-color: $grey-10-80;
+  --newtab-card-background-color: $grey-90;
+  --newtab-inner-box-shadow-color: $grey-60;
```

Comment:
I'm still looking through this, but are there any conceptually related uses of a color that are used for more than one variable? I don't think I spotted any yet, but for example if `--newtab-text-primary-color` and `--newtab-textbox-color` were the same, there could be a local scss $var to share the two. But then, why would we have two different css --var? (I suppose the css var could exist if say dark shared a color but light had different colors -- but then again, why are they not consistent between the two themes?)